cmd/oceancenter/main.go

```diff
@@ -0,0 +1,155 @@
+/*
+AUTHORS
+  Alan Noble <alan@ausocean.org>
+
+LICENSE
+  Copyright (C) 2024 the Australian Ocean Lab (AusOcean)
+
+  This file is part of Ocean Center. Ocean Center is free software: you can
+  redistribute it and/or modify it under the terms of the GNU
+  General Public License as published by the Free Software
+  Foundation, either version 3 of the License, or (at your option)
+  any later version.
+
+  Ocean Center is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with Ocean Center in gpl.txt.  If not, see
+  <http://www.gnu.org/licenses/>.
+*/
+
+// Ocean Center is a cloud service for remove device management, including:
+// 
+// - device software installation
+// - device software upgrades
+// - device enabling and disabling
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"sync"
+
+	"github.com/ausocean/cloud/model"
+	"github.com/ausocean/openfish/datastore"
+)
+
+const (
+	projectID = "oceancenter"
+	version   = "v0.1.0"
+)
+
+var (
```

Comment:
We could avoid these globals if we created an ```oceanCenter``` struct with these fields + methods ```indexHandler```, ```installHandler``` etc then created a value of this struct in ```func main```. Then we would just register the methods with ```http```.